File: yeongseong/7week/부족한 금액 계산하기.js
function solution(price, money, count) {
    let totalcost = 0;
    for (let i = 1; i <= count; i++){
        totalcost += price * i;
    }
    let shortage = totalcost - money; // 모자란 긍액
    if (shortage > 0){ // 모자른 금액
        return shortage;
    }else{ // 모자르지 않은 금액
        return 0;
    }
}

File: dongmin/week7/부족한 금액 계산하기.java
class Solution {
    public long solution(int price, int money, int count) {
        long totalPrice=0;
        for (int i = 1; i <= count; i++) {
            totalPrice+=price*i;        //총합 가격을 곱해서
        }
        return money>=totalPrice?0:totalPrice-money;        //return
    }
}